tabs/ansible/bigip/blueprint.yaml

```diff
@@ -3,7 +3,7 @@ tosca_definitions_version: cloudify_dsl_1_4
 description: Cloudify Redis blueprint. Deploys Redis service with Ansible Cloudify plugin.
 
 imports:
-  - https://raw.githubusercontent.com/cloudify-community/cloudify-catalog/6.4.0-build/tabs/utilities/eaas/custom_types.yaml
+  - https://repository.cloudifysource.org/cloudify/blueprints/6.4/custom_types.yaml
```

Comment:
When is the file uploaded to S3, is it done manual, or we have an automation for it?